packages/ecs-gpu-init/ecs-gpu-init.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Initialize ECS GPU config
+Before=ecs.service
+After=configured.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ecs-gpu-init
+
+[Install]
+WantedBy=preconfigured.target
+RequiredBy=ecs.service
```

Comment:
Hmm, why `After=configured.target` if we also have `WantedBy=preconfigured.target`? I'd expect us to finish creating the file either in preconfigured or in configured.